errors/errorHandlers.js

```diff
@@ -0,0 +1,20 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+  if (["22P02", "23502", "42601"].includes(err.code)) {
+    res.status(400).send({ msg: "Bad Request" });
+  } else next(err);
+};
```

Comment:
You may want to make the error messages more bespoke depending on the sql error code.